subiquity/client/controllers/welcome.py

```diff
@@ -37,11 +37,13 @@ async def make_ui(self):
 
     def run_answers(self):
         if 'lang' in self.answers:
-            self.done(self.answers['lang'])
+            self.done((self.answers['lang'], ""))
 
-    def done(self, code):
+    def done(self, lang):
+        (code, native) = lang
```

Comment:
I'm not going to remember what this means, would you do a quick comment showing what the values would look like in intended usage?

Reply:
Done.

Reply:
Kind of the problem here is that 'native' is not at all clear.  I was hoping to list an example something like `('de_DE.UTF-8', 'Deutsch')`, that the code is the locale value that we wish to set and that native refers to the translated display name of the language.

My goal here is that the next reader doesn't have to backtrack this all the way to the languagelist parser to know what the native value looks like.

Reply:
I see your point. Thanks for clarifying it further.